.ci/magician/vcr/tester.go

```diff
@@ -50,10 +50,10 @@ type logKey struct {
 type Tester struct {
 	env            map[string]string           // shared environment variables for running tests
 	rnr            ExecRunner                  // for running commands and manipulating files
-	logBucket      string                      // GCS bucket name to store logs
-	cassetteBucket string                      // GCS bucket name to store cassettes
```

Comment:
seems like maybe this move was included unintentionally?

Reply:
I wanted these to be in a logical order. I'm flipping `cassetteBucket` with `logBucket` now to be consistent with `cassettePaths` and `logPaths` and moving them back up so that `baseDir` is together with all the `*Path(s)`.

I'm also switching the order of parameters for `NewTester` to put cassettes before logs, since cassettes come first.

Reply:
wfm